go.mod

```diff
@@ -18,11 +18,11 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/MANTRA-Chain/cosmos-sdk v0.53.3-v5-mantra-1
 
 	// Direct tag link: https://github.com/MANTRA-Chain/evm/tree/mantra/v0.4.x_main
-	github.com/cosmos/evm => github.com/MANTRA-Chain/evm v0.0.0-20250814022355-d7ef57b65371
-	github.com/cosmos/evm/evmd => github.com/MANTRA-Chain/evm/evmd v0.0.0-20250814022355-d7ef57b65371
+	github.com/cosmos/evm => github.com/MANTRA-Chain/evm v0.0.0-20250820051844-dd50630e41fb
+	github.com/cosmos/evm/evmd => github.com/MANTRA-Chain/evm/evmd v0.0.0-20250820051844-dd50630e41fb
 
 	// branch: release/1.16
-	github.com/ethereum/go-ethereum => github.com/cosmos/go-ethereum v0.0.0-20250806193535-2fc7571efa91
+	github.com/ethereum/go-ethereum => github.com/yihuang/go-ethereum v1.10.21-0.20250820050041-210d5222d7fd
```

Comment:
https://github.com/cosmos/evm/pull/490/commits/1f6268b715dfc3c5bd787868df8a81aa8124baae
@mmsqe seems we can switch back to cosmos's go-ethereum  fork